conda/third_party/dali_libtar/recipe/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "dali-libtar" %}
```

Comment:
Add license? We have it in yml for ffmpeg

Reply:
Or is this copied from conda?

Reply:
Copied from https://github.com/conda-forge/libtar-feedstock.

Reply:
How about their license? Do we need any Acknowledgments?

Reply:
Added